user/routes/users.js

```diff
@@ -183,11 +183,11 @@ function create_new_result_set (req, session_id) {
     `INSERT INTO users_result_sets ` +
     `(set_querying_user_id, set_results_as_of, set_session_id, ` +
     `set_rownum, ` +
-    `id, name, versionkey) ` +
+    `id, name, tou, versionkey) ` +
```

Comment:
nope, not how we do it. see TERMS array. Esp since you didn't add the column to result_set